### PR TITLE
New tutorials

### LINKED ISSUE #400: use basic weldx file examples in documentation
some of our tutorials reuse code to generate the same basic layout (like a default V-Groove workpiece, a basic CSM instance etc.)
I think we should provide and load these basic examples as files that get created (and explained) in their own tutorials to slim down the feature tutorials to cover their essential topics

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

Adds a set  of new tutorials as discussed [here](https://github.com/BAMWelDX/weldx/discussions/455)

## Related Issues

Closes #400 

## Checks

- [ ] remove `single_pass_weld.wx`
- [ ] updated CHANGELOG.md
- [ ] updated tests
- [ ] updated doc/
- [ ] update example/tutorial notebooks
- [ ] update manifest file

### CODE REVIEW

**Comment by vhirtham** [comment]
I created this branch as a base branch for all the upcoming tutorials. I will make a separate PR for each tutorial targetting this branch so that we can discuss the tutorials individually.

**Comment by vhirtham** [comment]
@CagtayFabry @marscher I converted the test file in this branch to `0.5.0` by simply loading the `0.4.0` based file from the last conference and writing it back to disc. Interestingly, I can't open the file in the notebook of the first tutorial (see PR #556 ). I get the error:

~~~
pint.errors.UndefinedUnitError: 'Δ' is not defined in the unit registry
~~~

To my understanding, this is probably caused by the pint short notations. But before I continue searching for the problem, can you (@marscher ) confirm that you get this error too and that it isn't just a wrong setup on my computer? You need to change the file extension in the corresponding notebook cell back to `wx`. I am currently using the "old" file to continue working on the tutorial.

**Comment by marscher** [comment]
I receiving the same error.

**Comment by marscher** [comment]
```
ipdb> input_string
'Δ°C'
```

**Comment by CagtayFabry** [comment]
interesting, the notation is created by pint so wondering why it cannot load it

in the meantime I suggest changing the serialization notation style for pint units back to 'long' format here
https://github.com/BAMWelDX/weldx/blob/1ea8be3927106c7518d3ab731751d65027764120/weldx/tags/unit/pint_quantity.py#L58-L60

this should give the long notation again
```python
return f"{obj:.}"
```

**Comment by vhirtham** [comment]
> interesting, the notation is created by pint so wondering why it cannot load it
> 
> in the meantime I suggest changing the serialization notation style for pint units back to 'long' format here
> https://github.com/BAMWelDX/weldx/blob/1ea8be3927106c7518d3ab731751d65027764120/weldx/tags/unit/pint_quantity.py#L58-L60
> 
> this should give the long notation again
> 
> ```python
> return f"{obj:.}"
> ```

I will try that for now. I couldn't quickly pinpoint the field that was causing that since the python stack was cut at some point. I haven't done any deeper research since then

**Comment by CagtayFabry** [comment]
I have asked about handling this notation with pint here https://github.com/hgrecco/pint/issues/1390

File: tutorials/util.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+def download_tutorial_input_file():
+    from urllib.request import urlretrieve
+
+    url = (
+        "https://github.com/BAMWelDX/IIW2021_AA_CXII/blob/weldx_0.5.0"
+        "/single_pass_weld.weldx?raw=true "
+    )
+    sha256sum = "29e4f11ef1185f818b4611860842ef52d386ad2866a2680257950f160e1e098a"
```
Comment (CagtayFabry) [inline tutorials/util.py:14]: can we reuse this in `conf.py`? The code looks identical?

**Comment by CagtayFabry** [comment]
can you give a quick update on what is missing here @vhirtham ?

File: tutorials/util.py (hunk 1 above)
Comment (vhirtham) [reply tutorials/util.py:14]: Think so, this was part of the PR from @marscher  IIRC

**Comment by vhirtham** [comment]
> can you give a quick update on what is missing here @vhirtham ?

Nothing missing, we just didn't want to include it in the last release

File: tutorials/util.py (hunk 1 above)
Comment (marscher) [reply tutorials/util.py:14]: It is identical. Once we have mode the tutorials to the docs dir, we do not need this dupe any more.

**Review by CagtayFabry — APPROVED** [review]
👍 

just need to update the changelog entry

**Review by marscher — APPROVED** [review]

**Pull request merged**